src/main/resources/templates/spg/ADDRESS_TYPE_CHANGED-event.ftl

```diff
@@ -0,0 +1,17 @@
+"addressTypeChange" : {
```

Comment:
This file could use a reformat- consistency in colon-space structure and some indentation on hash objects.

Reply:
we can clean this up when we are not in a rush :P